cmd/rancherd/auth/auth.go

```diff
@@ -4,7 +4,10 @@ import (
 	"context"
 	"fmt"
 
+	restclient "k8s.io/client-go/rest"
+
 	"github.com/pkg/errors"
+	v3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
```

Comment:
We try to avoid import rancher library in rancherd. Can you use the dynamic client to update cluster?